plugin/plaintextmodel.cpp

```diff
@@ -24,6 +24,8 @@
 #include <QThread>
 #include <silicatheme.h>
 
+#include "uchardet/src/uchardet.h"
```

Comment:
Would be nice if c++ side was agnostic of the library placement, i.e. include as in how the lib is commonly used, build system setting up the flags so that proper directions are used for lookup.

Reply:
> Would be nice if c++ side was agnostic of the library placement, i.e. include as in how the lib is commonly used, build system setting up the flags so that proper directions are used for lookup.
Well actually you are right